tests/Mindee.UnitTests/MindeeClientTest.cs

```diff
@@ -4,6 +4,7 @@
 using Mindee.Pdf;
 using Mindee.Product.Custom;
 using Mindee.Product.Generated;
+using Mindee.Product.InternationalId;
```

Comment:
A bit late, but it's a shame we can't use invoice splitter for this, since it was the base for most of our async tests

Reply:
Wanted to test on LLM type